test/syscalls/BUILD

```diff
@@ -426,6 +426,7 @@ syscall_test(
 
 syscall_test(
     size = "medium",
+    tags = ["container"],
```

Comment:
One of test cases fails:

```
[ RUN      ] ProcPidMem.DifferentUserAttached
third_party/gvisor/test/syscalls/linux/proc.cc:764: Failure
Value of: syscall(105, kNobody)
Expected: not -1 (success)
  Actual: -1 (of type long), with errno PosixError(errno=22 Invalid argument)
```

The idea to run the whole proc_test in a container doesn't look right. I think you need to find out what test cases are needed to be run in isolated environments and move only them to a separate test.

Reply:
Thanks for the note! I moved the tests to a separate file `proc_defaults`. Since this also applies to https://github.com/google/gvisor/pull/6552 and https://github.com/google/gvisor/pull/6551, both were updated and rebased over this branch.